src/components/StartGame/StartGame.jsx

```diff
@@ -15,7 +15,9 @@ export class StartGame extends React.Component {
 
     render() {
         return (
-            <Button cy="start-game" handleClick={this.startGame} text="Start game" />
+            <div>
```

Comment:
why wrap in a div? are you just trying to make it `display: block` ?

Reply:
that's outdated